### PR TITLE
Use chainer.as_variable

### PR DESCRIPTION
**Pull request by muupan**
We no longer need define our own function to wrap an array by a variable as `chainer` has `as_variable` since v4 (https://docs.chainer.org/en/v4.0.0/reference/generated/chainer.as_variable.html).

### CODE REVIEW

**Comment by toslunar** [comment]
Do the args require grads?

**Comment by muupan** [comment]
Good point. It is difficult to guarantee that we won't compute gradient wrt `mean` or `var` of `GaussianDistribution`, so maybe we should not use `chainer.as_variable` here...

**Comment by muupan** [comment]
Some facts
- `ACER` computes gradient wrt Distribution.params via `backward`, not `chainer.grad`, so it can be affected by `requires_grad=False`.
- Currently `chainer.grad` can compute gradient even when `requires_grad=False`, but this is a bug of Chainer and will be fixed (https://github.com/chainer/chainer/issues/5974)
